scripts/powershell/publish-to-npm.ps1

```diff
@@ -0,0 +1,142 @@
+param (
```

Comment:
We should put this script directly in the JS repo as I don't expect anyone else is going to be using it.